src/applications/sign-in-changes/components/AccountSwitch.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { LoginButton } from '~/platform/user/exportsFile';
```

Comment:
Only use one of the imports for LoginButton (I think your mix-matching them)
- `@department-of-veterans-affairs/platform-user/LoginButton` OR
- `~/platform/user/authentication/components/LoginButton`